frontend/src/metabase/dashboard/components/DashboardButtonTooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ReactNode } from "react";
+import { Text, Tooltip } from "metabase/ui";
+
+type DashboardButtonTooltipProps = {
+  children: ReactNode;
+  label: string;
+};
+
+export function DashboardButtonTooltip({
+  children,
+  label,
+}: DashboardButtonTooltipProps) {
+  return (
+    <Tooltip
+      py="0.6rem"
+      px="0.75rem"
+      bg="bg-black"
+      offset={4}
+      label={
+        <Text c="inherit" size="sm" fw={700}>
+          {label}
+        </Text>
+      }
+      withArrow
+      arrowSize={10}
+    >
+      {children}
+    </Tooltip>
+  );
+}
```

Comment:
Update dashboard header button tooltips to use the new Mantine `Tooltip` component

Reply:
Nice! 😍 QQ: did you have to apply all of these styles to make the tooltips look exactly as before?

Reply:
I actually got this from somewhere else where someone converted the old tool-tip to use the new Mantine tooltip (https://github.com/metabase/metabase/blob/master/frontend/src/metabase/dashboard/components/DashboardEmbedHeaderButton/DashboardEmbedHeaderButton.tsx#L39-L51)